src/sleep.ts

```diff
@@ -0,0 +1,5 @@
+export function sleep(ms: number) : Promise<void> {
+    return new Promise<void>((resolve) => {
+        setTimeout(resolve, ms);
+    });
+}
```

Comment:
is this last line normal?

Reply:
@khenidak Don't understand this comment.  The last line is a closing brace to match the opening brace of the `sleep` function...?